plaso/engine/profilers.py

```diff
@@ -73,13 +85,13 @@ def IsSupported(cls):
 
   def Start(self):
     """Starts the profiler."""
-    filename = '{0:s}-{1:s}.csv.gz'.format(
+    filename = '{0!s}-{1:s}.csv.gz'.format(
```

Comment:
Why support anything else than a string here?

Reply:
Changed the initialization of _FILENAME_PREFIX instead